src/entities/drawer/ui/DrawingBoard.tsx

```diff
@@ -50,6 +50,8 @@ type Props = {
 const DrawingBoard: FC<Props> = props => {
   const { value, onResult, onStarted, width, isDrawingActive, onLog } = props;
 
+  const [lastPath, setLastPath] = useState<DrawLine[]>([]);
```

Comment:
The approach with lastPath looks obsolete and it also overlaps the existing solution